src/components/LicenseFilters/LicenseFilters.js

```diff
@@ -173,17 +176,21 @@ export default class LicenseFilters extends React.Component {
   renderEndDateFilter = () => {
     return <DateFilter
       activeFilters={this.props.activeFilters}
+      allowNull
```

Comment:
https://github.com/folio-org/ui-agreements/pull/739#discussion_r563998922

Reply:
Change made, to showNoDateCheckbox (switched from showNoEndDateCheckbox since this component is covers both start and end dates)